test/performance/comm/low-level/arrayTransfer.ml-execopts

```diff
@@ -0,0 +1,11 @@
+--xferMem=512   --runSecs=1 --op=opPut # array-put-512B
+--xferMem=4096  --runSecs=1 --op=opPut # array-put-4096B
+--xferMB=1      --runSecs=1 --op=opPut # array-put-1M
+--xferMB=1024   --runSecs=1 --op=opPut # array-put-1G
+--memFraction=4 --runSecs=1 --op=opPut # array-put-FR4
```

Comment:
Transferring 1/4 of memory in 1 second seems unachievable in practice.  If that's so then the runs across different transfer sizes will vary in length and things won't be as comparable as we'd like.  That may not matter much, though, since only the 1/4 mem ones will be out of range.  The rest should be able to get done in 1 second.

Reply:
Yeah, you're right. It doesn't actually finish in a second, this really just limits it to 1 trial at least on large-memory cray compute nodes. On a 128GB compute node it takes ~4 seconds. That said I have no problem with the elapsed time being different between versions since we're graphing the rate.